specification/cognitiveservices/data-plane/AzureOpenAI/authoring/preview/2023-04-01-preview/images.json

```diff
@@ -0,0 +1,348 @@
+                  "Internal",
+                  "TooManyRequests",
+                  "Unauthorized",
+                  "NotFound",
+                  "OperationNotInTerminalState",
+                  "BlockedImages",
+                  "RequestTimout"
+               ],
+               "description": "The error code."
+            },
+            "details": {
+               "type": "array",
+               "items": {
+                  "$ref": "#/definitions/InnerError"
+               }
+            }
+         },
+         "required": [
+            "message",
+            "code"
+         ]
+      },
+      "ErrorResponse": {
+         "type": "object",
+         "properties": {
+            "error": {
+               "$ref": "#/definitions/Error"
+            }
+         },
+         "required": [
+            "error"
+         ]
+      },
+      "ImageResult": {
+         "type": "object",
+         "description": "The image url if successful, and an error otherwise.",
+         "properties": {
+            "url": {
+               "type": "string",
+               "description": "The image url.",
+               "example": "https://www.contoso.com"
+            },
+            "error": {
+               "$ref": "#/definitions/Error"
+            }
+         }
+      },
+      "OperationResult": {
+         "type": "array",
+         "description": "The result data of the operation, if successful",
+         "items": {
+            "$ref": "#/definitions/ImageResult"
+         }
+      },
+      "OperationResponse": {
+         "type": "object",
+         "properties": {
+            "id": {
+               "type": "string",
+               "description": "The ID of the operation.",
+               "example": "59c7e2ea-7450-4e85-9762-55c78727f4c7"
+            },
+            "created": {
+               "type": "integer",
+               "format": "int64",
+               "description": "The unix timestamp when the operation was created.",
+               "example": "1676540381"
+            },
+            "last_action": {
+               "type": "integer",
+               "format": "int64",
+               "description": "The unix timestamp for when the current state was entered.",
+               "example": "1676540381"
+            },
+            "expires_at": {
+               "type": "integer",
+               "format": "int64",
+               "description": "The unix timestamp when the operation expires.",
+               "example": "1676540381"
+            },
+            "data": {
+               "$ref": "#/definitions/OperationResult"
+            },
+            "status": {
+               "$ref": "#/definitions/Status"
+            },
+            "error": {
+               "$ref": "#/definitions/Error"
+            }
+         },
+         "required": [
+            "id",
+            "status",
+            "created",
+            "last_action"
+         ]
+      },
+      "ImageResolution": {
```

Comment:
>Resolution [](http://example.com/codeflow?start=12&length=10)

Dimensions?

Reply:
I actually renamed it to "Size" as this name is also used by OpenAI.